src/Umbraco.Web/CompositionExtensions.cs

```diff
@@ -19,21 +20,21 @@
 
 // the namespace here is intentional -  although defined in Umbraco.Web assembly,
 // this class should be visible when using Umbraco.Core.Components, alongside
-// Umbraco.Core's own CompositionExtensions class
+// Umbraco.Core's own IUmbracoBuilderExtensions class
 
 // ReSharper disable once CheckNamespace
 namespace Umbraco.Web
 {
     /// <summary>
-    /// Provides extension methods to the <see cref="Composition"/> class.
+    /// Provides extension methods to the <see cref="IUmbracoBuilder"/> class.
     /// </summary>
-    public static class WebCompositionExtensions
+    public static class WebIUmbracoBuilderExtensions
```

Comment:
Super minor naming. I think leaving the "I" out of interface extensions is preferable in most cases, so just `WebUmbracoBuilderExtensions` ?

Reply:
Someone got trigger happy with find and replace :(

Reply:
resolved via #9451